### DISCUSSION TITLE
Por API, usuario administrador no puede encontrar tasks

### DISCUSSION THREAD

**Discussion by seiturra** [Cotalker/documentation#504] [4. Bug Reports] [answered]
Buenos días,

Tenemos una implementación donde un usuario bot con permisos de administrador necesita encontrar una tarea (sin estar en ella) para hacer un cambio en un dato de su asset.schemaInstance. Para esto usamos el endpoint
```
GET /api/tasks/${taskGroupId}/task/${taskId}
```
Cuando hago la llamada con el Authorization header de un usuario que está en la task, lo encuentra. Cuando lo hago con el bot, devuelve un status 500 con body: {"error":"Error: Task not found","status":false}

¿Esto puede ser un cambio reciente en el endpoint? Hasta donde tengo entendido esta implementación estaba funcionando hace unos meses.

Gracias!

**Comment by guillevalin** [comment, marked as answer]
@SeIturra Si lo intentas como usuario, no la encontrará porque no está en la tarea como participante. Sin embargo, si el token de usuario que hace la llamada de la API, tiene permisos de administrador, puedes utilizar el header ```admin: true``` y podrá buscarla como administrador.

**Reply by seiturra** [reply]
Probado y aceptado.

Gracias!
